slack/web/client.py

```diff
@@ -2154,3 +2154,66 @@ def views_publish(
         else:
             kwargs.update({"view": view})
         return self.api_call("views.publish", json=kwargs)
+
+    def workflows_stepCompleted(
```

Comment:
Running `python setup.py validate` updates `AsyncWebClient` for you. 

As the `black` formatter behaves differently in Travis CI env, checking the absence of the execution is not enabled in CI builds yet. https://github.com/slackapi/python-slackclient/blob/v2.8.2/.travis.yml#L11-L12

Reply:
As discussed, you'll run this command for me this time around due to the reformatting issues occurring